test/cli/errors/errors.out

```diff
@@ -1,150 +1,150 @@
+    15 |    raise arg # raise is defined by stdlib
             ^^^^^^^^^
     https://github.com/sorbet/sorbet/tree/master/rbi/core/kernel.rbi#L850: Method `Kernel#raise (overload.1)` has specified `arg0` as `String`
      850 |        arg0: String,
                   ^^^^
   Got Integer originating from:
-    test/cli/errors/errors.rb:12:
-    12 |  def foo(arg)
+    test/cli/errors/errors.rb:13:
+    13 |  def foo(arg)
                   ^^^
 
-test/cli/errors/errors.rb:18: Expected `Integer` but found `String("foo")` for argument `arg` https://srb.help/7002
-    18 |    foo("foo")
+test/cli/errors/errors.rb:19: Expected `Integer` but found `String("foo")` for argument `arg` https://srb.help/7002
+    19 |    foo("foo")
             ^^^^^^^^^^
-    test/cli/errors/errors.rb:11: Method `ComplexError#foo` has specified `arg` as `Integer`
-    11 |  sig {params(arg: Integer).returns(NilClass)}
+    test/cli/errors/errors.rb:12: Method `ComplexError#foo` has specified `arg` as `Integer`
+    12 |  sig {params(arg: Integer).returns(NilClass)}
                       ^^^
   Got String("foo") originating from:
-    test/cli/errors/errors.rb:18:
-    18 |    foo("foo")
+    test/cli/errors/errors.rb:19:
+    19 |    foo("foo")
                 ^^^^^
 
-test/cli/errors/errors.rb:32: Expected `Integer` but found `T.nilable(Integer)` for argument `a` https://srb.help/7002
-    32 |    bar(a)
+test/cli/errors/errors.rb:33: Expected `Integer` but found `T.nilable(Integer)` for argument `a` https://srb.help/7002
+    33 |    bar(a)
             ^^^^^^
-    test/cli/errors/errors.rb:35: Method `ErrorLines#bar` has specified `a` as `Integer`
-    35 |  sig {params(a: Integer).returns(Integer)}
+    test/cli/errors/errors.rb:36: Method `ErrorLines#bar` has specified `a` as `Integer`
+    36 |  sig {params(a: Integer).returns(Integer)}
                       ^
   Got T.nilable(Integer) originating from:
-    test/cli/errors/errors.rb:25:
-    25 |  def main(foo)
+    test/cli/errors/errors.rb:26:
+    26 |  def main(foo)
           ^^^^^^^^^^^^^
-    test/cli/errors/errors.rb:28:
-    28 |          nil
+    test/cli/errors/errors.rb:29:
+    29 |          nil
                   ^^^
-    test/cli/errors/errors.rb:30:
-    30 |          2
+    test/cli/errors/errors.rb:31:
+    31 |          2
                   ^
   Autocorrect: Use `-a` to autocorrect
-    test/cli/errors/errors.rb:32: Replace with `T.must(a)`
-    32 |    bar(a)
+    test/cli/errors/errors.rb:33: Replace with `T.must(a)`
+    33 |    bar(a)
                 ^
 Errors: 4
-test/cli/errors/errors.rb:4: Unable to resolve constant `MissingConstant` https://srb.help/5002
-     4 |    MissingConstant
-            ^^^^^^^^^^^^^^^
-    https://github.com/sorbet/sorbet/tree/master/rbi/stdlib/optparser.rbi#L743: Did you mean: `OptionParser::MissingArgument`?
-     743 |  class MissingArgument < OptionParser::ParseError; end
-            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This is the error I was trying to get rid of. Unfortunately it also had the side effect of changing every line number.